schema/Retail/RET11/on_search.ts

```diff
@@ -475,6 +483,35 @@ export const FnBonSearchSchema = {
                             maximum_value: {
                               type: 'string',
                             },
+                            tags: {
+                              type: 'array',
+                              items: {
+                                type: 'object',
+                                properties: {
+                                  code: {
+                                    type: 'string',
+                                    enum: ['range', 'default_selection'],
+                                  },
+                                  list: {
+                                    type: 'array',
+                                    items: {
+                                      type: 'object',
+                                      properties: {
+                                        code: {
+                                          type: 'string',
+                                          enum: ['lower', 'upper', 'value', 'maximum_value'],
```

Comment:
exact validation of lower/upper for range and value/maximum_value for default_selection need to be checked in business validations